examples/with-nextjs-and-clerk-auth/src/components/ThemeSelect/ThemeSelect.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FC, SyntheticEvent, useRef, useState } from 'react';
+
+import {
+  UilSun,
```

Comment:
Haven't you checked what the is console showing us?

<img width="1190" alt="image" src="https://github.com/team-monite/monite-sdk/assets/725645/fa1a6be3-aa5f-4275-8b26-152403338ddb">

Reply:
I am aware of this issue, it is from the icon library which I am required to use to fulfil the goals of this project.